### PR TITLE
fix(cli): respect custom `buildDir` option for `tasks` sub command

### PR DESCRIPTION
**Pull request by TMBL-DEV**
updated list & run to get a nitro instance and grab the buildDir from the nitro options and at the close the nitro instance.

### 🔗 Linked issue
[#2634 Task cli forces .nitro as buildDir](https://github.com/unjs/nitro/issues/2634)

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

#### Changes
In run.ts & list.ts from the task command i used `createNitro()` to have a nitro instance so i could access the options.
i use the `nitro.options.builDir` to replace the hardcoded `".nitro"` buildDir that is passed to the runTask function.
at the end of the command run function i also close the the nitro instance that i created.

#### Why
Now if your `buildDir` in the `nitro.config.ts `is not the default .nitro directory instead of getting an error it uses the correct buildDir and your command will run.
This will also make sure that the task cli works with nuxt, because nuxt uses the the buildDir .nuxt.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/cli/commands/task/list.ts [hunk 1]
```diff
@@ -16,13 +16,17 @@ export default defineCommand({
   },
   async run({ args }) {
     const cwd = resolve((args.dir || args.cwd || ".") as string);
-    const tasks = await listTasks({ cwd, buildDir: ".nitro" });
+    const nitro = await createNitro({ rootDir: cwd });
```
Comment (pi0) [inline src/cli/commands/task/list.ts:19]: I think you can use lighter `loadConfig` to load nitro config only

Comment (TMBL-DEV) [reply src/cli/commands/task/list.ts:19]: I'll take a look at it today

Comment (TMBL-DEV) [reply src/cli/commands/task/list.ts]: i used the loadOptions() because i couldn't find an example on the loadConfig function.
I hope that's okay.

**Comment by TMBL-DEV** [comment]
@pi0 is this good?

**Review by pi0 — APPROVED** [review]

**Pull request merged**